modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -13,7 +13,7 @@ def initialize
     end
 
     def perform
-      load_page(current_page: 0)
+      load_page(current_page: -1)
```

Comment:
Indexing on `vaforms/search_action.asp` starts at 0, so the first page should be `-1` in order to account for the increment inside of `load_page`. Otherwise, we increment to 1, then hit the `unless current_page == 1` condition, which removes the query params to get the first page, but the next iteration of the loop goes to page 3 instead of page 2.